docs/products/kafka/howto/connect-java.rst

```diff
@@ -0,0 +1,178 @@
+Java samples for testing Aiven for Apache Kafka®
+=================================================
+
+The sample Java code in this article demonstrates how to connect to your Aiven for Apache Kafka service and send messages into a topic.
```

Comment:
We're not sending messages in a topic anymore :D 
```suggestion
The sample Java code in this article demonstrates how to connect to your Aiven for Apache Kafka service as producer or consumer of messages.
```

Reply:
I removed that part completely :)